parts/windowsProvision.ps1

```diff
@@ -13,6 +13,12 @@ $PACKAGES = @{
         "url" = "https://github.com/PowerShell/Win32-OpenSSH/releases/download/v7.7.2.0p1-Beta/OpenSSH-Win64.zip"
         "local_file" = Join-Path $PACKAGES_DIRECTORY "OpenSSH-Win64.zip"
     }
+    "AzureDCAP" = @{
+        "url" = "https://www.nuget.org/api/v2/package/Azure.DCAP.Windows/0.0.1"
+        "local_file" = Join-Path $PACKAGES_DIRECTORY "azure.dcap.windows.0.0.1.nupkg"
+        "renamed_file" = Join-Path $PACKAGES_DIRECTORY "azure.dcap.windows.0.0.1.zip"

```

Comment:
We should eventually add another field here (and for all other downloads), like "SHA256_hash", which verifies the integrity of the downloaded file after the file is downloaded. This isn't in the scope of this PR, so I'll file an issue here on GitHub to do that

Reply:
I agree let's work on it on the following PR.

Reply:
I've filed a new issue for this. Feel free to ignore this comment in the context of this PR :)